README.md

```diff
@@ -1,4 +1,6 @@
-# RAML to HTML
+# RAML to HTML - Updated for RAML 1.0
+
+**IMPORTANT: This is a development branch and should be considered experimental/ pre-alpha**
```

Comment:
Is this really needed? Everything not in master / on NPM is experimental right :)